docs/Onboarding/index.md

```diff
@@ -74,41 +74,50 @@
 1. On "Register your device" select Set-up and login with your MS account with TAP, and when registration is complete click "Done"
 1. Give Google Services permissions (location, diagnostic data etc...) and click on "Accept"
 1. Agree to End user License Agreement and click "Next"
-1. Configure Microsoft Authenticator app (login with company account and then register authenticator with company)
+1. Configure Microsoft Authenticator app (login with Microsoft company account of user that is going to be using device and then register authenticator with company)
 1. Login to Microsoft Defender (if already logged in with company account account should be present, just select it) and give it requested  permissions, wait for the scan to complete
 1. Open Outlook and Teams app, check if auto logged in (if already logged in with company account account should be present, just select it)
 
 ## Onboarding Device - Android Personally-owned devices with work profile
-
+1. Before continuing please login  with the desired Google personal account.
 1. Install Intune Company Portal from Google Play store, [https://play.google.com/store/apps/details?id=com.microsoft.windowsintune.companyportal](https://play.google.com/store/apps/details?id=com.microsoft.windowsintune.companyportal)
-1. Login to Company Portal (using Microsoft Entra with TAP) and within Company portal:
+1. Login to Company Portal with Microsoft company account of user that is going to be using device (using Microsoft Entra with TAP) and within Company portal:
     1. Create work profile
     1. Activate work profile
     1. On last step resolve any warnings and Confirm Device settings
-1. Configure Microsoft Authenticator app (login with company account and then register authenticator with company)
-1. Login to Microsoft Defender (if already logged in, company account account should be present, just select it)
+1. Configure Microsoft Authenticator app (login with Microsoft company account of user that is going to be using device)
+1. Login to Microsoft Defender and give it required permisions (if already logged in, company account account should be present, just select it)
 1. Open Outlook and Teams app, check if auto logged in (if already logged in with company account account should be present, just select it)
 
+
 ## Onboarding Device - iOS web based enrollment
 
 Reference: [Set up web based device enrollment](https://learn.microsoft.com/en-us/mem/intune-service/enrollment/web-based-device-enrollment-ios)
-
-1. Open Safari and go to https://portal.manage.microsoft.com/enrollment/webenrollment/ios. Sign in with your work or school account (with TAP)
+1. Before continuing please ensure that you have logged in the device with Apple account.
```

Comment:
similiar like on android, move to Note, in next step, do not have special step for this check